tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es6",
+    "outDir": "out",
+    "lib": ["es6", "dom"],
+    "sourceMap": true,
+    // "forceConsistentCasingInFileNames": true,
+    // "noImplicitAny": true,
```

Comment:
Why were these options changed? I don't think that's desired.

Reply:
My bad. The compiler options are different in `server`, `client`, and `html_views`, and I weakened checks by trying to unify them: fixed now.